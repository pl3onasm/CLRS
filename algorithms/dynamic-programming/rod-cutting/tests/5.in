5

0 12.96 15.56 25.81 35.45 40.15 50.42 60.44 64.44 70.35 80.2 85.4 90.43 100.76 115.84 130.46 145.32 160.95 175.51 195.63 220.61 245.91 270.66 300.33 331.72 370.77 405.58 450.37 500.93 551.55 605.7 660.4 720.02 805.77 990.87 1050.22 1200.86 1310.03 1450.50