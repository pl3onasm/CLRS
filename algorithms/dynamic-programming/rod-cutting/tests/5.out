Rod length: 5
Maximum revenue: 64.80
Optimal cuts: 1, 1, 1, 1, 1
